server/accounts.go

```diff
@@ -278,6 +278,8 @@ func (a *Account) shallowCopy() *Account {
 	}
 	// JetStream
 	na.jsLimits = a.jsLimits
+	// Server config account limits.
+	na.limits = a.limits
```

Comment:
Did we miss that for jwt before?

Reply:
No JWT fills in when it process the claim, it pulls apart the limits from the JWTs and places them into the account.

https://github.com/nats-io/nats-server/blob/acc_config_limits/server/accounts.go#L3168